Sprint-3/implement/get-angle-type.js

```diff
@@ -10,18 +10,58 @@
 // When the angle is exactly 90 degrees,
 // Then the function should return "Right angle"
 
```

Comment:
I believe the assignment required you to define the function only once, rather than multiple times, as a single function should be able to inform you of the angle type regardless of the number. Do you know of any ways you'd be able to do that while satisfying all the conditionals.